### ISSUE TITLE
The navigation of the interface is not simple and increases user cognitive load.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5220]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Navigation in the chat region is not easy. Cognitive users will find it difficult in navigation, remembering or understanding when to use the [up/down arrow] and when to [Tab] is frustrating. The user can easily move out of the ‘Help me choose’ window without even knowing it. 

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=tru

### How to reproduce the issue?

1. Open [https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=true](https://wwwppe.microsoft.com/en-us/surface) in Edge. 
2. Start NVDA. 
3. Turn on the NVDA ‘Speech viewer”. 
4. Navigate the interface and listen to or read the content.

### What do you expect?

The navigation of the interface should be simple and clear.

### What actually happened?

The navigation of the interface increases user cognitive load. Determining what text and sections are in the chat history is unclear. Remembering or understanding when to use the [up/down arrow] and when to [Tab] is frustrating. The user can easily move out of the ‘Help me choose’ window without even knowing it.  

### MAS reference

https://aka.ms/MAS1.3.1

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by OEvgeny** [comment]
We're removing this popup as we already follow well known keyboard shortcuts #5218
